pkgs/applications/networking/remote/aws-workspaces/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, lib
+, makeWrapper, dpkg, fetchurl, autoPatchelfHook
+, curl, kerberos, lttng-ust, libpulseaudio, gtk3, openssl_1_1, icu, webkitgtk, librsvg, gdk-pixbuf, libsoup, glib-networking
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aws-workspaces";
+  version = "3.1.0.813";
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
+
+  dontStrip = true;
```

Comment:
```suggestion
```
Is there a reason why we should not strip?

Reply:
Yep, a very good reason actually: it crashes at startup when I do it :smile: 
`Failed to create CoreCLR, HRESULT: 0x80004005`

Reply:
Can you add a comment with that information?

Reply:
Done in 4b1675c784bccf503d8623eccb4b47050dc37ba7

Reply:
BTW this is the case with most (all?) .NET applications. Haven't looked into why.